CONTRIBUTING.md

```diff
@@ -39,8 +39,13 @@ Before creating an issue please make sure that it was not already reported.
 #### Code
 
 1) Create a new branch based on `develop` branch.
+    * Optional: create and enable virtualenv:
```

Comment:
I used a virtualenv here as I didn't want to pollute my local machine with extra packages. Thought it might be useful for others, too.